database/seeds/OrganizationRolePermissionSeeder.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+use Illuminate\Database\Seeder;
+
+class OrganizationRolePermissionSeeder extends Seeder
+{
+    /**
+     * Run the database seeds.
+     *
+     * @return void
+     */
+    public function run()
+    {
+        $organizationPermissionTypes = DB::table('organization_permission_types')->select('id', 'feature')->get();
+
+        foreach ($organizationPermissionTypes as $organizationPermissionType) {
+            $adminRole = DB::table('organization_role_types')->where('name', 'admin')->first();
+
+            DB::table('organization_role_permissions')->insert([
+                'organization_role_type_id' => $adminRole->id,
+                'organization_permission_type_id' => $organizationPermissionType->id
+            ]);
+
+            $courseCreatorRole = DB::table('organization_role_types')->where('name', 'course_creator')->first();
+
+            if (in_array($organizationPermissionType->feature, ['Project', 'Playlist', 'Activity', 'Team', 'Group', 'Search'])) {
+                DB::table('organization_role_permissions')->insert([
+                    'organization_role_type_id' => $courseCreatorRole->id,
+                    'organization_permission_type_id' => $organizationPermissionType->id
+                ]);
+
+                $selfRegisteredRole = DB::table('organization_role_types')->where('name', 'self_registered')->first();
+
+                DB::table('organization_role_permissions')->insert([
+                    'organization_role_type_id' => $selfRegisteredRole->id,
+                    'organization_permission_type_id' => $organizationPermissionType->id
+                ]);
+            }
+
+
```

Comment:
Remove additional line breaks

Reply:
Removed new line between "course creator" and "member".